src/Resources/views/storefront/page/search/search-pagelet.html.twig

```diff
@@ -21,39 +19,9 @@
         {% if page.header.extensions.flConfig.filterPosition is constant('\\FINDOLOGIC\\FinSearch\\Findologic\\FilterPosition::LEFT') %}
             {% set sidebarClasses = 'has--sidebar' %}
```

Comment:
Where is this variable even used?

Reply:
Used in Shopware templates, but it's also set there so I don't see any reason to do it here.